internal/repository/profile_repository.go

```diff
@@ -77,13 +78,24 @@ func (profileStore *ProfileStore) ListProfiles(ctx context.Context) (values []dt
 	}
 	rows, err := profileStore.db.Query(ctx, sql, args...)
 	if err != nil {
-		zap.S().Error("error executing create project insert query:", err)
+		zap.S().Error("error executing list project insert query:", err)
 		return []dto.ListProfiles{}, err
 	}
 
 	for rows.Next() {
 		var value dto.ListProfiles
-		rows.Scan(&value.ID, &value.Name, &value.Email, &value.YearsOfExperience, &value.PrimarySkills, &value.IsCurrentEmployee)
+		var isCurrentEmployee int
+		err := rows.Scan(&value.ID, &value.Name, &value.Email, &value.YearsOfExperience, &value.PrimarySkills, &isCurrentEmployee)
+		if err != nil {
+			zap.S().Error("error scanning row:", err)
+			return []dto.ListProfiles{}, err
+		}
+
+		if isCurrentEmployee == 1 {
```

Comment:
handle it in service layer and use `map[bool]string{}{true:"Yes",false:"No"}`

try to keep independent db level no any service layer code, later on may be we need to change repo type pgSql -> mySql

Reply:
Done!